modules/openshift-developer-cli-openshift-cluster-objects.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/odo-architecture.adoc
+
+[id="openshift-cluster-objects_{context}"]
+= OpenShift cluster objects
+
+== Init Containers
+Init Containers are specialized containers that run before the application container starts and configure the necessary environment for the application containers to run. Init containers can have files that app images do not have, for example setup scripts. Init containers always run to completion and the application container does not start if any of the init containers fails.
+
+The Pod created by {odo-title} executes two init containers:
+. The copy-supervisord init container.
+. The copy-files-to-volume init container.
+
+=== `copy-supervisord`
```

Comment:
Is this necessary to be in backticks? In the following sentence you do not - just be consistent whichever way.

Reply:
Changed everything to backticks.